### PR TITLE
^10.6.0

### PR DESCRIPTION
**Pull request by MadDataScience**
## Related Issues

- Update airgap.js-types
- Make VIEW_STATE_TO_MESSAGE partial so to exclude IABViewState

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
   },
   "devDependencies": {
     "@monaco-editor/react": "^4.4.5",
-    "@transcend-io/airgap.js-types": "^10.4.2",
+    "@transcend-io/airgap.js-types": "^10.6.0",
```
Comment (linh-transcend) [inline package.json:47]: I don't think I understand why the version jump is so large? Not sure if that makes sense.

Comment (MadDataScience) [reply package.json:47]: 10.4.2 was published [2 months ago](https://github.com/transcend-io/airgap.js-types/pkgs/npm/airgap.js-types/versions). There have been 8 versions between 10.4.2 and 10.6.0 (including 6 point versions of 10.5) but no one bothered to update consent-manager-ui until now. I suppose it wasn't necessary, but this is a breaking change.

Comment (anotherminh) [reply package.json:47]: nit: it's not a breaking change in functionality/the API of this library, it's just a big type change..

Comment (MadDataScience) [reply package.json:47]: True. It's a breaking-tsc change, but not a breaking-functionality change.

Comment (MadDataScience) [reply package.json:47]: Actually, to be fair, this (type) actually broke in [10.5.0](https://github.com/transcend-io/airgap.js-types/pull/91) with the introduction of `TCF_EU` to `ViewState`. The change from 10.5 to 10.6 is, in fact, unrelated to the fix in BottomMenu.

**Review by anotherminh — APPROVED** [review]

**Pull request merged**